index.bs

```diff
@@ -1462,3 +1485,278 @@ The [=remote end steps=] with |command parameters| are:
+     [=/browsing contexts=] which are [=child browsing contexts=] of |parent|.
+
+  1. Let |contexts info| be a list.
+
+  1. For each |context| of |child contexts|:
+
+    1. Let |info| be the result of [=get the browsing context info=] given
+       |context|, |depth|, and |max depth|.
+
+    1. Append |info| to |contexts info|
+
+  1. Return |contexts info|
+
+</div>
+
+### Commands ### {#module-browsingContext-commands}
+
+#### The browsingContext.getTree Command ####  {#command-browsingContext-list}
+
+The <dfn export for=commands>browsingContext.getTree</dfn> command returns a
+tree of all browsing contexts that are children of the given context, or all
+top-level contexts when no parent is provided.
+
+<dl>
+   <dt>Command Type</dt>
+   <dd>
+      <pre class="cddl remote-cddl">
+      BrowsingContextGetTreeCommand = {
+        method: "browsingContext.getTree",
+        params: BrowsingContextGetTreeParameters
+      }
+
+      BrowsingContextGetTreeParameters = {
+        ?maxDepth: uint,
+        ?parent: BrowsingContext,
+      }
+      </pre>
+   </dd>
+   <dt>Return Type</dt>
+   <dd>
+    <pre class="cddl local-cddl">
+        BrowsingContextGetTreeResult = {
+          contexts: [*BrowsingContextInfo]
+        }
+    </pre>
+   </dd>
+</dl>
+
+<div algorithm="remote end steps for browsingContext.getTree">
+The [=remote end steps=] with |command parameters| are:
+
+  1. Let the |parent id| be the value of the <code>parent</code> field of
+     |command parameters| if present, or null otherwise.
+
+  1. Let |max depth| be the value of the <code>maxDepth</code> field of |command
+     parameters| if present, or 0 otherwise.
+
+  1. Let |depth| be 0.
+
+  1. Let |contexts| be the result of [=get the ancestor browsing contexts=],
+     given |parent id|, |depth|, and |max depth|.
+
+  1. Let |body| be a [=map=] matching the <code>BrowsingContextListResult</code>
+     production, with the <code>contexts</code> field set to |contexts|.
+
+  1. Return [=success=] with data |body|.
+
+</div>
+
+### Events ### {#module-contexts-events}
+
+#### The browsingContext.contextCreated Event #### {#event-browsingContext-contextCreated}
+
+<dl>
+   <dt>Event Type</dt>
+   <dd>
+      <pre class="cddl local-cddl">
+        BrowsingContextCreatedEvent = {
+         method: "browsingContext.contextCreated",
+         params: BrowsingContextInfo
+       }
+      </pre>
+   </dd>
+</dl>
+The [=remote end event trigger=] is:
+
+<div algorithm="remote end event trigger for browsingContext.contextCreated">
+When the [=create a new browsing context=] algorithm is invoked, after the
+[=active document=] of the browsing context is set, run the following steps:
+
+ 1. Let |context| be the newly created browsing context.
+
+ 1. Let |related contexts| be a set containing the [=parent browsing context=]
+    of |context|, if that is not null, or an empty set otherwise.
+
+ 1. Let |params| be the result of [=get the browsing context info=] given
```

Comment:
What should be used for depth and max depth here?

Reply:
@k7z45 note that this was resolved by https://github.com/w3c/webdriver-bidi/pull/62/commits/53d42f73673ed55cbe29dc16687b217d25eff3f2 with *depth* 0 and *max depth* 1, causing "get the descendent browsing contexts" to return null in its first step.